src/ensembl/src/shared/components/new-selected-species/selected-species-common.scss

```diff
@@ -0,0 +1,29 @@
+@import 'src/styles/common';
+
+$selectedSpeciesBorderRadius: 20px;
+$selectedSpeciesPadding: 6px 20px;
+$deleteButtonPadding: 8px 20px 8px 28px;
+
+@mixin selected-species-common {
```

Comment:
this mixin (and the file itself) probably don't make sense on their own anymore; previously, there were simple selected species and focusable selected species, both of which were using styles from this common file; but now there's only one component; so these styles can move over to `SelectedSpecies.scss`

Reply:
We need to create a ticket to create new mixin that can be used for round buttons (as decided in the code review)

Reply:
https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-774

No need to create a new one 😄